.gitignore

```diff
@@ -3,7 +3,7 @@
 .*.swp
 .*.swo
 .idea/
-result
+/result
```

Comment:
why? please revert

Reply:
One of the added packages is called "result" and it gets gitignored with the current setting

Reply:
Result dirs can be created in other places than the git root dir

Rename the result package dir to `python-result`

Reply:
ok, that works for me

Reply:
it does not even seem to be part of this PR, so let's not do any change at all to gitignore then